.github/workflows/ci-iov.yml

```diff
@@ -57,4 +68,46 @@ jobs:
       - name: rust-api-unit-tests
         run: ci_run zk test rust-api
 
+      - name: start-services (rskj, postgres, zk)
+        run: |
+          docker-compose -f docker-compose-runner.yml down
+          docker-compose -f docker-compose-runner.yml pull
+          docker-compose -f docker-compose-runner.yml up --build -d rskj postgres zk 
+          ci_run sccache --start-server &
+
+      - name: run zk init
+        run: |
+          ci_run zk
+          ci_run zk init
+
+      - name: restart dev-liquidity-token-watcher and dev-ticker
+        run: docker-compose -f docker-compose-runner.yml restart dev-liquidity-token-watcher dev-ticker
+
+      - name: run-services
+        run: |
+          ci_run zk server core &>server.log &
+          ci_run zk server api &>api.log &
+          ci_run sleep 10
+          ci_run zk dummy-prover run &>dummy_prover.log &
+          ci_run sleep 100
+
+      - name: server-integration-tests
+        run: ci_run zk test i server
 
+      - name: api-integration-tests
+        run: |
+          ci_run zk test i api
+          ci_run zk test i api-docs
+
+      - name: rust-sdk-integration-tests
+        run: ci_run zk test i rust-sdk
+
+      - name: withdrawal-helpers-integration-tests
+        run: ci_run zk test i withdrawal-helpers
+
+      - name: Show logs
+        if: always()
```

Comment:
Do we need this `if` here?

Reply:
yes we do, you can read the definition here. https://docs.github.com/en/actions/learn-github-actions/expressions#always.